padcli/provider/repo.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type Repository interface {
-	Get(context.Context, Cluster) (RepoConfig, error)
+	Get(context.Context, Cluster, string) (RepoConfig, error)
```

Comment:
can we add a comment saying what `string` is? ok yes, it's the repo, but does it also include the tag? or just the repo location?